docs/index.md

```diff
@@ -37,9 +37,9 @@ Pulse provides push-based integrations with your workflow using the following al
 
     Pulse currently supports the following "one-click" integrations that simplify the process of setting up your workflows to send data to Pulse:
 
-    -   **[GitHub integration](one-click-integrations/github-integration.md):** reports changes and deployments
-    -   **[PagerDuty integration](one-click-integrations/pagerduty-integration.md):** reports incidents
-    -   **[Jira integration](one-click-integrations/jira-integration.md):** reports lead and cycle time
+    -   **[GitHub integration](one-click-integrations/github-integration.md)** and **[Bitbucket integration](one-click-integrations/bitbucket-integration.md)**: reports changes and deployments
```

Comment:
```suggestion
    -   **[GitHub integration](one-click-integrations/github-integration.md)** or **[Bitbucket integration](one-click-integrations/bitbucket-integration.md)**: reports changes and deployments
```

change to `or` because while we're not going to block users from configuring both git providers, it's not a use case that we planned for